src/components/StudyPane/InfoPane/Motif/RootBlock.tsx

```diff
@@ -1,62 +1,79 @@
+  const [textColorLocal, setTextColorLocal] = useState(availableDescendant.textColor? availableDescendant.colorFill: DEFAULT_TEXT_COLOR);
   const [selected, setSelected] = useState(false);
 
+  const matchExpandedWordsColorScheme = () => {
+    let match = descendants.every((dsd) => {
+      if (ctxExpandedStanzas.at(dsd.stanzaId as number) === false || ctxExpandedStrophes.at(dsd.stropheId as number) === false) {
+        return true;
+      }
+      const matchesBorderColor = !dsd.borderColor || dsd.borderColor === availableDescendant.borderColor;
+      const matchesColorFill = !dsd.colorFill || dsd.colorFill === availableDescendant.colorFill;
+      const matchesTextColor = !dsd.textColor || dsd.textColor === availableDescendant.textColor;
+
+      return matchesBorderColor && matchesColorFill && matchesTextColor;
+    });
+    return match;
+  }
+
   useEffect(() => {
-    const rootBlockColor = ctxRootsColorMap.get(descendants[0].strongNumber);
-    if (rootBlockColor) {
-      setColorFillLocal(rootBlockColor.colorFill);
-      setTextColorLocal(rootBlockColor.textColor);
-    }
-  }, [ctxRootsColorMap]);
-  
+    const displayedDescendant = descendants.find((dsd) => {
+      if (ctxExpandedStanzas.at(dsd.stanzaId as number) === true && ctxExpandedStrophes.at(dsd.stropheId as number) === true) {
+        return true;
+      }
+    })
+    let stubHebWord: HebWord = descendants[0]; // used to satisfy type of availableDescendants, but not used because setSelected does not run unless displayedDescendant is not falsy;
+    const availableDescendant = displayedDescendant || stubHebWord;
+    setDescendantDisplayed(availableDescendant)
+  }, [ctxExpandedStanzas, ctxExpandedStrophes])
+
   useEffect(() => {
-    const colorFill = descendants[0].colorFill;
-    const isSameColorFill = descendants.every((x) => x.colorFill === colorFill);
-    if (!isSameColorFill) {
-       setColorFillLocal(DEFAULT_COLOR_FILL);
-    } else if (colorFill != colorFillLocal) { setColorFillLocal(colorFill); }
-
-    const textColor = descendants[0].textColor;
-    const isSameTextColor = descendants.every((x) => x.textColor === textColor);
-    if (!isSameTextColor) {
-       setTextColorLocal(DEFAULT_TEXT_COLOR);
-    } else if (textColor != textColorLocal) { setTextColorLocal(textColor); }
-  }, [descendants]);
+    const rootBlockColor = ctxRootsColorMap.get(availableDescendant.strongNumber);
+    const matchedColorScheme = matchExpandedWordsColorScheme();
+    if (rootBlockColor || matchedColorScheme) {
+      setColorFillLocal(rootBlockColor? rootBlockColor.colorFill: availableDescendant.colorFill);
+      setBorderColorLocal(rootBlockColor? rootBlockColor.borderColor: availableDescendant.borderColor);
+      setTextColorLocal(rootBlockColor? rootBlockColor.textColor: availableDescendant.textColor);
+    }
+    if (!matchedColorScheme) {
+      setColorFillLocal(DEFAULT_COLOR_FILL);
```

Comment:
We want matching color fill in root word even if it doesn't have matching border/text color